server/models/data.js

```diff
@@ -0,0 +1,237 @@
+const urllib = require('urllib')
+
+let state = {
+  isLoaded:false,
+  playersIds:0
+}
+let allPlayersData = []
+let teamsLoaded = {isLoaded:false}
+let playersIds = {current:0}
+
+const getAllPlayers = () => {
```

Comment:
this should be at the end of this file - first declare all the variable, and then declare functions if needed